src/Ensconce.Cli/ApplicationInteraction.cs

```diff
@@ -141,7 +176,63 @@ private static IEnumerable<ServiceDetails> GetServicesInstalledInDirectory(strin
                     PathName = svc.GetPropertyValue("PathName").ToString(),
                     DisplayName = svc.GetPropertyValue("DisplayName").ToString(),
                     ServiceName = svc.GetPropertyValue("Name").ToString()
-                });
+                })
+                .ToList();
+        }
+
+        private static List<ProcessDetails> GetProcessesRunningInDirectory(string directory)
+        {
+            if (!directory.EndsWith(@"\"))
```

Comment:
Path.DirectorySeparatorChar

Reply:
ahhh yeah, i did only copy this...but good to fix it whilst i'm there :)